src/Nut/Init.php

```diff
@@ -38,4 +48,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return 0;
     }
+
+    /**
+     * Archive distributions have the site bundle files installed as .dist
+     * files to prevent the real files being overridden. If the .dist file
+     * exists, but the original doesn't then we should rename them.
+     */
+    private function updateDistBundles()
+    {
+        $fs = $this->app['filesystem']->getFilesystem('root');
+        $paths = ['.bolt.yml', 'composer.json', 'composer.lock', 'src/Site/SiteBundleLoader.php'];
```

Comment:
Don't you prefer:
```
$paths = [
    '.bolt.yml',
    'composer.json',
    'composer.lock',
    'src/Site/SiteBundleLoader.php',
];
```

Reply:
Haha … Payback is the proverbial … yeah, probably. I don't know that this is **exactly** what we want either way, just trying to keep the conversation moving right now :wink: